File: c++/Assignment_Pattern/Assignment2/Q8.cpp
#include <iostream>
using namespace std;
int main(){
    int n;
    cout<<"Enter no. of lines : ";
    cin>>n;
    for(int i=1;i<=n;i++){
        for(int j=1;j<=(2*n-1);j++){
            int b=j;
            if(j>n) b=2*n-j;
            if((i+b)==(n+1))
                cout<<i;
            else
                cout<<" ";
        }
        cout<<endl;
    }
    return 0;
}

File: java/Practice/Pattern_Printing/Q8.java
import java.util.Scanner;

public class Q8 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter number of lines : ");
        int n=sc.nextInt();

        for(int i=1;i<=n;i++){
            for(int s=n-i;s>=1;s--){
                System.out.print(" ");
            }
            for(int j=1;j<=i;j++){
                System.out.print("* ");
            }
            System.out.println();
        }

        sc.close();
    }
}
